src/Client.php

```
@@ -136,7 +137,7 @@ public function query(array $command, ?string $db = null): stdClass|array|int
             '$db' => $db ?? $this->database,
         ]);
 
-        $sections = BSON\fromPHP($params);
+        $sections = BSON\fromPHPWithOptions($params);
```

PHPStan (2.1.17)

Function MongoDB\BSON\fromPHPWithOptions not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(function.notFound)